templates/flows/flow_editor.haml

```diff
@@ -359,6 +359,38 @@
       confirmation.open = true;      
     }
 
+    {% if google_translate %}
+      async function handleSystemTranslate(code, text) {
+        const response = await fetch("https://translation.googleapis.com/language/translate/v2?key={{google_translate}}", {
```

Comment:
Here we would need to have the API calls to Google/DeepL on the backend side, that way we'll be able to hide the API keys.